oauthproxy.go

```diff
@@ -1091,6 +1092,26 @@ func checkAllowedGroups(req *http.Request, s *sessionsapi.SessionState) bool {
 	return false
 }
 
+// checkAllowedEmails allow email restrictions based on the `allowed_emails`
+// querystring parameter
+func checkAllowedEmails(req *http.Request, s *sessionsapi.SessionState) bool {
+	allowedEmails := extractAllowedEntities(req, "allowed_emails")
+	if len(allowedEmails) == 0 {
+		return true
+	}
+
+	allowed := false
+
+	for email := range allowedEmails {
+		if email == s.Email {
+			allowed = true
```

Comment:
NIT: Why not just return `true` here & return `false` at the end if you don't have any matches?

Reply:
Ah, you're right, I don't know why I did it like this. Should I do a separate pull to change this since this one is already merged?

Reply:
If you could do the minor fix, that would be good